approved/ip_xact_sub_block.xml

```diff
@@ -3,18 +3,18 @@
   <spirit:vendor>Origen</spirit:vendor>
   <spirit:library>Origen</spirit:library>
   <spirit:name>d_ip_ana_test_annex_syn</spirit:name>
-  <spirit:version></spirit:version>
+  <spirit:version/>
   <spirit:memoryMaps>
     <spirit:memoryMap>
-      <spirit:name></spirit:name>
+      <spirit:name>atx</spirit:name>
```

Comment:
@chrishume is this really needed? When the subblock is imported, atx gets initialized as a sub_block within the sub block, so it becomes dut.atx2.atx